data/systemd/snap.mount.in

```diff
@@ -0,0 +1,14 @@
+# ensure @SNAP_MOUNT_DIR@ MS_SHARED so that lxd works correctly (LP: #1668659)
```

Comment:
The sentence would read slightly better as:

_# Ensure @SNAP_MOUNT_DIR@ is mounted "shared" so lxd works correctly (LP: #1668759)._

Reply:
Done